lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -1025,4 +1052,5 @@ def ensemble_cve(kymotracks):
         method="ensemble cve",
         unit=cve_based[0].unit,
         _unit_label=cve_based[0].unit,
+        localization_variance_variance=ensemble_localization_var_var,
```

Comment:
Naming (see above).

Reply:
`variance_of_localization_variance` and `var_of_localization_var`

Reply:
👍